online_test/settings.py

```diff
@@ -64,11 +65,17 @@
 
 # Database
 # https://docs.djangoproject.com/en/1.6/ref/settings/#databases
-
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.{0}'.format(
```

Comment:
Why is sqlite the default db. Since the mariadb image that we pull is set with default username and password ```root```, as defined in ```docker-compose.yml```, why not just add that to the ```.env.sample``` file. So that a newbie who builds it without changing the ```.env.sample``` file will get to use the mysql db out of the box.

Reply:
That is just a sane default in case the .env file is not present. Refer to https://github.com/henriquebastos/python-decouple/blob/master/README.rst#attention-with-undefined-parameters